### PR TITLE
Platform 3190

### PR DESCRIPTION
**Pull request by sczerwinski-wikia**

### CODE REVIEW

File: extensions/wikia/Recirculation/WikiRecommendations.class.php [hunk 1]
```diff
@@ -574,6 +574,24 @@ class WikiRecommendations {
 		],
 	];
 
+    const STAGING_RECOMMENDATIONS = [
```
Comment (Grunny) [inline extensions/wikia/Recirculation/WikiRecommendations.class.php]: Should be tabs, not spaces throughout.

File: extensions/wikia/Recirculation/WikiRecommendations.class.php [hunk 2]
```diff
@@ -591,20 +609,21 @@ class WikiRecommendations {
...
-		if ( empty( $wgDevelEnvironment ) ) {
-			$recommendations = self::RECOMMENDATIONS['en'];
+	public static function getRecommendations( $contentLanguage ) {
+		global $wgWikiaEnvironment;
 
-			if ( array_key_exists( $contentLanguage, self::RECOMMENDATIONS ) ) {
-				$recommendations = self::RECOMMENDATIONS[$contentLanguage];
-			}
-			shuffle( $recommendations );
+        if ( $wgWikiaEnvironment == WIKIA_ENV_STAGING ){
```
Comment (Grunny) [inline extensions/wikia/Recirculation/WikiRecommendations.class.php]: Both should use strict equals `===` per conventions.

**Review by Grunny — CHANGES_REQUESTED** [review]

File: extensions/wikia/Recirculation/WikiRecommendations.class.php (hunk 1 above)
Comment (sczerwinski-wikia) [reply extensions/wikia/Recirculation/WikiRecommendations.class.php]: Yeah, copy&paste in PHPStorm

File: extensions/wikia/Recirculation/WikiRecommendations.class.php (hunk 2 above)
Comment (sczerwinski-wikia) [reply extensions/wikia/Recirculation/WikiRecommendations.class.php]: Done

File: extensions/wikia/Recirculation/WikiRecommendations.class.php (hunk 1 above)
Comment (sczerwinski-wikia) [reply extensions/wikia/Recirculation/WikiRecommendations.class.php]: Done

File: extensions/wikia/Recirculation/WikiRecommendations.class.php [hunk 3]
```diff
@@ -591,20 +609,21 @@ class WikiRecommendations {
...
+			$recommendations = self::STAGING_RECOMMENDATIONS;
+		} elseif ( $wgWikiaEnvironment === WIKIA_ENV_DEV ) {
+			$recommendations = self::DEV_RECOMMENDATIONS;
+		} else {
 			$recommendations = self::RECOMMENDATIONS['en'];
 
 			if ( array_key_exists( $contentLanguage, self::RECOMMENDATIONS ) ) {
 				$recommendations = self::RECOMMENDATIONS[$contentLanguage];
-			}
+		}
```
Comment (Grunny) [inline extensions/wikia/Recirculation/WikiRecommendations.class.php]: This bracket should be indented an extra level.

**Review by Grunny — CHANGES_REQUESTED** [review]

File: extensions/wikia/Recirculation/WikiRecommendations.class.php (hunk 3 above)
Comment (sczerwinski-wikia) [reply extensions/wikia/Recirculation/WikiRecommendations.class.php]: Already done in amend to Duplicated dev recommendation fix, I haven't noticed the push was rejected

**Review by Grunny — APPROVED** [review]

**Pull request merged**